skyportal/handlers/api/spectrum.py

```diff
@@ -573,7 +573,7 @@ def get(self, obj_id):
         if normalization is not None:
             if normalization == "median":
                 for s in return_values:
-                    norm = np.median(s["fluxes"])
+                    norm = np.median(np.abs(s["fluxes"]))
```

Comment:
Need to avoid the case where the median is exactly zero (which will lead to a divide by zero error)
```suggestion
                    norm = np.median(np.abs(s["fluxes"]))
                    norm = norm if norm != 0.0 else 1e-20
```